third-party/dnd-wiki/classes/class-weapon-master.xml

```diff
@@ -0,0 +1,518 @@
+		<description>
+			<p>At 3rd level, you choose the path that you walk as a Weapon master. Choose Combat Mastery, Tactic Mastery, or Evasive Mastery all detailed at the end of the class description. The pathway you choose grant you features at 3rd level and again at 7th, 10th, 15th, and 18th level.</p>
+		</description>
+		<sheet>
+			<description></description>
+		</sheet>
+		<setters>
+			<set name="sourceUrl">https://www.dandwiki.com/w/index.php?title=Weapon_Master_(5e_Class)&amp;oldid=1131376#Martial_Archetype</set>
+		</setters>
+		<rules>
+			<select type="Archetype" name="Weapon Master Martial Archetypes" supports="Weapon Master Martial Archetypes" />
+		</rules>
+	</element>
+	<element name="Ability Score Improvement (Weapon Master)" type="Class Feature" source="D&amp;D Wiki" id="ID_WIKI_CLASS_FEATURE_WEAPON_MASTER_ABILITYSCOREIMPROVEMENT">
+		<description>
+			<p>When you reach 4th level, and again at 8th, 12th, 16th, and 19th level, you can increase one ability score of your choice by 2, or you can increase two ability scores of your choice by 1. As normal, you can't increase an ability score above 20 using this feature.</p>
+		</description>
+		<sheet display="false" />
+		<setters>
+			<set name="sourceUrl">https://www.dandwiki.com/w/index.php?title=Weapon_Master_(5e_Class)&amp;oldid=1131376#Ability_Score_Increase</set>
+		</setters>
+		<rules>
+			<!-- <grant type="Class Feature" id="ID_INTERNAL_CLASS_FEATURE_ASI_4_WEAPON_MASTER" level="4" requirements="!ID_INTERNAL_OPTION_ALLOW_FEATS" /><grant type="Class Feature" id="ID_INTERNAL_CLASS_FEATURE_ASI_8_WEAPON_MASTER" level="8" requirements="!ID_INTERNAL_OPTION_ALLOW_FEATS" /><grant type="Class Feature" id="ID_INTERNAL_CLASS_FEATURE_ASI_12_WEAPON_MASTER" level="12" requirements="!ID_INTERNAL_OPTION_ALLOW_FEATS" /><grant type="Class Feature" id="ID_INTERNAL_CLASS_FEATURE_ASI_16_WEAPON_MASTER" level="16" requirements="!ID_INTERNAL_OPTION_ALLOW_FEATS" /><grant type="Class Feature" id="ID_INTERNAL_CLASS_FEATURE_ASI_19_WEAPON_MASTER" level="19" requirements="!ID_INTERNAL_OPTION_ALLOW_FEATS" /> -->
+			<select type="Class Feature" name="Improvement Option (Weapon Master 4)" supports="Improvement Option,Weapon Master,4" level="4" />
+			<select type="Class Feature" name="Improvement Option (Weapon Master 8)" supports="Improvement Option,Weapon Master,8" level="8" />
+			<select type="Class Feature" name="Improvement Option (Weapon Master 12)" supports="Improvement Option,Weapon Master,12" level="12" />
+			<select type="Class Feature" name="Improvement Option (Weapon Master 16)" supports="Improvement Option,Weapon Master,16" level="16" />
+			<select type="Class Feature" name="Improvement Option (Weapon Master 19)" supports="Improvement Option,Weapon Master,19" level="19" />
+		</rules>
+	</element>
+	<element name="Extra Attack" type="Class Feature" source="D&amp;D Wiki" id="ID_WIKI_CLASS_FEATURE_WEAPON_MASTER_EXTRA_ATTACK">
```

Comment:
You can add the new extra attack stat so it shows the number on the sheet (see fighter for example)